.github/workflows/chronos-howto-guides-test.yml

```diff
@@ -68,3 +68,50 @@ jobs:
         env:
           BIGDL_ROOT: ${{ github.workspace }}
           ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+
+      - name: Run tests for Chronos-benchmark-tool
+        shell: bash
+        run: |
+          if conda info --env | grep "chronos-benchmakr-tool-env"; then
+            source deactivate
+            conda remove -n chronos-benchmakr-tool-env -y --all
+          fi
+          conda create -n chronos-benchmakr-tool-env -y python==3.7.10 setuptools==58.0.4
+          source activate chronos-benchmakr-tool-env
+          pip install pytest==5.4.1
+          pip install prophet==1.1
+          pip install pmdarima==1.8.4
+          pip install tsfresh==0.17.0
+          pip install ray==1.9.2 ray[tune]==1.9.2 ray[default]==1.9.2
+          pip install pyarrow==6.0.1
+          pip install ConfigSpace==0.5.0
+          pip install optuna==2.10.1
+          pip install neural-compressor==1.11
+          pip install onnxruntime==1.11.1
+          pip install onnx==1.11.0
+          pip install onnxruntime-extensions==0.4.2
+          pip install onnxsim==0.4.8
+          pip install scipy==1.5.4
+          pip install cloudpickle
+          apt-get update
+          apt-get install patchelf
+          pip uninstall -y bigdl-friesian bigdl-friesian-spark3 bigdl-dllib bigdl-dllib-spark3 bigdl-orca pyspark bigdl-orca-spark3 bigdl-chronos bigdl-chronos-spark3 bigdl-nano bigdl-friesian bigdl-friesian-spark3
+          sed -i "s/pyspark==2.4.6/pyspark==3.1.3/g" python/dllib/src/setup.py
+          #sed -i "s/'bigdl-core=='+VERSION/'bigdl-core==2.1.0b20220811'/g" python/dllib/src/setup.py
+          bash python/dev/release_default_linux_spark313.sh default false false
+          bash python/nano/dev/build_and_install.sh linux default false pytorch --force-reinstall --no-deps -U
+          whl_name=`ls python/nano/dist/`
+          pip install python/nano/dist/${whl_name}[tensorflow,pytorch]
+          pip install  python/dllib/src/dist/bigdl_dllib_*-py3-none-manylinux1_x86_64.whl
+          pip install  python/orca/src/dist/bigdl_orca_*-py3-none-manylinux1_x86_64.whl
+          pip install  python/chronos/src/dist/bigdl_chronos_*-py3-none-manylinux1_x86_64.whl
+          export SPARK_LOCAL_HOSTNAME=localhost
+          export KERAS_BACKEND=tensorflow
+          export PYTHONPATH=$PYTHONPATH:/home/user/wangning/BigDL/python/chronos/src/
+          bash python/chronos/dev/test/run-benchmark-tool.sh
+          source deactivate
+          conda remove -n chronos-benchmakr-tool-env -y --all
```

Comment:
some of the installation in github action could be simplify, but we could do it in later PR rather than this since we are having a horrible proxy condition recently, if we could pass all the action, then we should merge it.

Reply:
OK